go.mod

```diff
@@ -183,3 +281,20 @@ replace (
 
 // Plugin kn-plugin-event from midstream
 replace knative.dev/kn-plugin-event => github.com/openshift-knative/kn-plugin-event v0.32.1-0.20220601003552-7c29d149ba16
+
+// Plugin kn-plugin-func from midstream
+replace (
+	github.com/cloudevents/sdk-go/sql/v2 => github.com/cloudevents/sdk-go/sql/v2 v2.0.0-20220930150014-52b12276cc4a
+	github.com/openshift/source-to-image => github.com/boson-project/source-to-image v1.3.2
+
+	k8s.io/api => k8s.io/api v0.23.5
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.23.4
+	k8s.io/apimachinery => k8s.io/apimachinery v0.23.5
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.23.4
+	k8s.io/client-go => k8s.io/client-go v0.23.5
+	k8s.io/code-generator => k8s.io/code-generator v0.23.5
+	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20220124234850-424119656bbf
+	knative.dev/func => github.com/openshift-knative/kn-plugin-func v1.1.3-0.20221021003942-828132c477de
```

Comment:
@lance FYI for the code freeze. The final version should land here.

E.g.:
```bash
knative.dev/func => github.com/openshift-knative/kn-plugin-func serverless-1.26.0
# That's resolving correct deps without downgrades/upgrades
go mod tidy -compat=1.17
go mod vendor
# That's a vendor cleanup, because we don't use hack/update-deps.sh
find vendor/ \( -name "OWNERS" -o -name "OWNERS_ALIASES" -o -name "BUILD" -o -name "BUILD.bazel" -o -name "*_test.go" \) -exec rm -f {} +
```